packages/react-devtools-shared/src/backend/fiber/renderer.js

```diff
@@ -1480,6 +1488,7 @@ export function attach(
         return true;
       case HostPortal:
       case HostText:
+      case ActivityComponent:
```

Comment:
So this is actually a good example for why we need this to be a separate wrapper. Because this should not be ignored. It should be visible in React DevTools because it's not an internal implementation detail but actually something the user renders.

The reason OffscreenComponent was hidden was because it's an internal of Suspense and it would be very noisy to show it everywhere but the Activity one we actually want to show.

So this should actually go into one of the other paths (and be filterable as a "type" in the filters).

This could be done as a follow up though.

Reply:
Good callout - fixed. I also found a few other places in devtools that I missed and fixed those too.